presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -101,7 +101,7 @@ tableElement
     ;
 
 columnDefinition
-    : identifier type (COMMENT STRING)?
```

Comment:
We are adding an alternative syntax for string literals. This should simply change the `STRING` token rather than replacing all of the usages (or worse, only some).